rest/data/project.go

```diff
@@ -259,6 +260,27 @@ func (pc *DBProjectConnector) UpdateProjectVars(projectId string, varsModel *res
 	return nil
 }
 
+func (pc *DBProjectConnector) UpdateProjectVarsByValue(toReplace string, replacement string) ([]*db.ChangeInfo, error) {
+	matchingProjects, err := model.GetVarsByValue(toReplace)
+	if matchingProjects == nil {
+		return nil, errors.Wrap(err, "no projects with matching value found")
+	}
+	changes := []*db.ChangeInfo{}
+	for _, project := range matchingProjects {
+		for key, val := range project.Vars {
+			if val == toReplace {
+				project.Vars[key] = replacement
+				change, err := project.Upsert()
```

Comment:
Are we assuming that there will only be one upsert per project (as opposed to a project containing the value 20 times and us doing 20 upserts on the same project)?

Reply:
the odds that a project has a value set multiple times is low enough that i'm not sure we need to consider it

Reply:
also we don't actually really use changes at all, so it would be reasonable to just do `_, err = project.Upser()` and only have the function return err